.changeset/cyan-turkeys-run.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/copyable': patch
+---
+
+Fixes Copyable Tooltip when used within a Modal
```

Comment:
Let's be a little clearer in the changeset: 

something like: Tooltips that now appear in association with Copyable component are no longer portaled, in order to support usage within a Modal